src/ssh-util/src/tunnel.rs

```diff
@@ -142,7 +142,7 @@ impl SshTunnelConfig {
             let local_port = Arc::clone(&local_port);
             async move {
                 scopeguard::defer! {
-                    info!(%tunnel_id, "terminating ssh tunnel");
+                    debug!(%tunnel_id, "terminating ssh tunnel");
```

Comment:
These were hard-won log lines from the SSHpocalypse earlier this year that were very intentionally added to make it possible to assess exactly where an SSH tunnel had failed. I tuned them carefully to avoid spam.  Do you have more context for exactly how they were spamming? I would be _really_ loathe to lose them.